src/endpoints/characters.js

```diff
@@ -74,12 +74,17 @@ async function writeCharacterData(inputFile, data, outputFile, request, crop = u
          * Read the image, resize, and save it as a PNG into the buffer.
          * @returns {Promise<Buffer>} Image buffer
          */
-        function getInputImage() {
-            if (Buffer.isBuffer(inputFile)) {
-                return parseImageBuffer(inputFile, crop);
-            }
+        async function getInputImage() {
+            try {
+                if (Buffer.isBuffer(inputFile)) {
+                    return await parseImageBuffer(inputFile, crop);
+                }
 
-            return tryReadImage(inputFile, crop);
+                return await tryReadImage(inputFile, crop);
+            } catch (error) {
+                console.log(`Failed to read image: ${inputFile}. Using a fallback image.`);
```

Comment:
Why hide the error? Might be good to still log it.
```suggestion
                console.warn(`Failed to read image: ${inputFile}. Using a fallback image.`, error);
```

Reply:
When I tried that it output a bunch of broken data, like it tried to log the entire binary blob converted to ASCII.

Reply:
God. Node console output not being able to log errors? Well, so be it.
You can still make this as a yellow warning at least.

Reply:
Yeah, I see the reason now. Fixed.